### PR TITLE
ICD11 Ingest - Data

### PR DESCRIPTION
**Pull request by joeflack4**
## Overview
Data files after running `build-mondo-ingest`

### Documentation
Was the documentation added/updated under `docs/`?
- [ ] Yes
- [x] No, updates to the docs were not necessary after careful consideration

### QC
Was the full pipeline run before submitting this PR using `sh run.sh make build-mondo-ingest` on this branch (after 
`docker pull obolibrary/odkfull:dev`), and no errors occurred?
- [x] Yes
- [ ] No, there are no functional (code-related) changes to the pipeline in the PR, so no re-run is necessary
   
### New Packages
Were any new *Python packages* added?
- [ ] Yes, [requirements.txt.full](https://github.com/INCATools/ontology-development-kit/blob/master/requirements.txt.full) was updated and an [ODK issue](https://github.com/INCATools/ontology-development-kit/issues/new) submitted
- [x] No

Were any other *non-Python packages* added?
- [ ] Yes, [Dockerfile](https://github.com/INCATools/ontology-development-kit/blob/master/Dockerfile) updated and an [ODK issue](https://github.com/INCATools/ontology-development-kit/issues/new) submitted
- [x] No

### PR Review and Conversations Resolved
Has the PR been sufficiently reviewed by at least 1 team member of the Mondo Technical team and all threads resolved?
- [ ] Yes

### CODE REVIEW

**Comment by joeflack4** [comment]
Lexmatch still not complete (https://github.com/monarch-initiative/mondo-ingest/pull/434#discussion_r1522352320)

File: docs/sources/icd11foundation.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ This data source in particular is the ICD11 foundation, not one of its lineariza
 
 **Homepage:** https://icd.who.int/
 
+<<<<<<< HEAD
```
Comment (matentzn) [inline docs/sources/icd11foundation.md:12]: Merge error

**Review by matentzn — COMMENTED** [review]
Apart from the missing icd11foundation lexmatch mapping, I also expected an `slurp/icd11foundation.tsv` file.

File: docs/sources/icd11foundation.md (hunk 1 above)
Comment (joeflack4) [reply docs/sources/icd11foundation.md:12]: Glad you noticed that. I thought I'd resolved all the conflicts in that file. Fixed!

File: src/ontology/mondo-ingest.Makefile [hunk 2]
```diff
@@ -159,6 +159,17 @@ $(COMPONENTSDIR)/icd10who.owl: $(TMPDIR)/icd10who_relevant_signature.txt | compo
 		remove -T config/properties.txt --select complement --select properties --trim true \
 		annotate --ontology-iri $(URIBASE)/mondo/sources/icd10who.owl --version-iri $(URIBASE)/mondo/sources/$(TODAY)/icd10who.owl -o $@; fi
 
+$(COMPONENTSDIR)/icd11foundation.owl: $(TMPDIR)/icd11foundation_relevant_signature.txt | component-download-icd11foundation.owl
```
Comment (joeflack4) [inline src/ontology/mondo-ingest.Makefile:162]: ## Build check passing
I reran and the following things I was trying to fix are now addressed:
- `slurp/icd11foundation.tsv` appears
- `lexmatch/unmapped_icd11foundation_lex.tsv` appears (among many other lexmatch files) and has over 1k lines (~6k)

As of my writing this, the build is still running. But I'll push again when the rest of it completes.

Comment (joeflack4) [reply src/ontology/mondo-ingest.Makefile:162]: @matentzn I reviewed this with Trish today. These two conditions checked above have been met for a couple weeks, and these are the conditions needed to be met to address this as success.

We're not merging this ("data PRs") for a variety of reasons; standard part of the mondo-ingest developer workflow.

FYI see how long it took me to run above, per your request.

**Pull request closed without merging**